tests/integration/components/ember-popper/events-enabled-test.js

```diff
@@ -9,12 +9,12 @@ moduleForComponent('ember-popper', 'Integration | Component | eventsEnabled', {
 
 test('sets eventsEnabled in the Popper instance', function(assert) {
   this.render(hbs`
-    <div class='parent' style='position: fixed; bottom: 0; height: 100px; width: 100vw;'>
-      {{#ember-popper placement='top' class='events-enabled'}}
+    <div class='parent' style='height: 100px; width: 100%;'>
+      {{#ember-popper placement='bottom' class='events-enabled'}}
```

Comment:
Now that we know Popper.js doesn't actually listen for resize events on the target, I'm wondering how this test works at all 🤔